### DISCUSSION TITLE
Questions and Ideas

### DISCUSSION THREAD

**Discussion by luffythink** [IntensiveCoLearning/Web3-URL#117] [Announcements]
大家可以将自己关于 Web3 URL 学习笔记的 「Questions and Ideas」 分享在 Discussions 各栏目：
- General，Chat about anything and everything here
- Ideas and Project Teams，Share ideas and form a project team.
- Q&A，Ask the community for help

**Comment by CHENFANGC** [comment]
[hello-web3 demo](https://github.com/ethstorage/hello-web3)  没有理解它输出的链接是https://0xedDC3d17a23eD3e1DeBfF90534eB54532755049F.3336.w3link.io/
为什么不是web3://……这种形式呢？

**Reply by luffythink** [reply]
是 web3:// 这种形式，web3://0xeddc3d17a23ed3e1debff90534eb54532755049f:3336 ，通过 [Firefox extension](https://addons.mozilla.org/en-US/firefox/addon/web3url/) 会被转译成https://0xeddc3d17a23ed3e1debff90534eb54532755049f.3336.w3link.io/

**Reply by CHENFANGC** [reply]
用的Chrome访问，为什么是火狐的扩展呢？

**Reply by luffythink** [reply]
Chrome 不是还不支持 web3:// 这种格式嘛。目前通过火狐的扩展就是用来支持 web3:// 直接访问呀。
